src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+    private static final String URL = "jdbc:mysql://localhost:3306/mate";
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "password3452";
+
+    public static Connection getConnection() {
+        try {
+            return DriverManager.getConnection(URL, USER_NAME, PASSWORD);
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't connect with data base", e);
+        }
+    }
+}

```

Comment:
Don't forget about empty lines.
Check all files.
```suggestion
}

```